benchmarks/src/main/java/org/questdb/PongMain.java

```diff
@@ -134,6 +134,10 @@ public void sendPong() {
     private static class PongRequestProcessor implements IORequestProcessor<PongConnectionContext> {
         @Override
         public boolean onRequest(int operation, PongConnectionContext context) {
+            if (operation == IOOperation.HEARTBEAT) {
```

Comment:
hearbeat can be handled more gracefully, for example instead of `context.getDispatcher().disconnect(context, DISCONNECT_REASON_UNKNOWN_OPERATION);` we can have something like `context.getDispatcher().handleDefault(operation, context);`

Reply:
`context.getDispatcher()` returns an interface, and I wouldn't bloat its API.